👨‍🌾 Warning in windows packaging jobs "ignoring '/INCREMENTAL' due to '/LTCG' specification"

## Bug report

**Required Info:**

- Operating System:
  - Windows 10
- Installation type:
  - source
- Version or commit hash:
  - starting from 0ebae0cea78ee63864aea8b70b2962e24f42cb69
- DDS implementation:
  - default
- Client library (if applicable):
  - rclpy

#### Steps to reproduce issue

There is a new warning in the windows packaging jobs:
```
   Building Custom Rule C:/ci/ws/src/ros2/rclpy/rclpy/CMakeLists.txt

  _rclpy_action.cpp

LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/LTCG' specification [C:\ci\ws\build\rclpy\_rclpy_action.vcxproj]

     Creating library C:/ci/ws/build/rclpy/RelWithDebInfo/_rclpy_action.lib and object C:/ci/ws/build/rclpy/RelWithDebInfo/_rclpy_action.exp

```
I think it [started](https://ci.ros2.org/job/packaging_windows/2026/msbuild/) right after #667 was merged.

---
https://github.com/osrf/buildfarmer/issues/161

--- a/rclpy/CMakeLists.txt
+++ b/rclpy/CMakeLists.txt
@@ -180,6 +180,7 @@ target_link_libraries(_rclpy_pybind11 PRIVATE
   rclpy_common
 )
 configure_build_install_location(_rclpy_pybind11)
+clean_windows_flags(_rclpy_pybind11)
 
 # Extension with code for Action servers and clients
 pybind11_add_module(_rclpy_action SHARED
@@ -192,6 +193,7 @@ target_link_libraries(_rclpy_action PRIVATE
   rcutils::rcutils
 )
 configure_build_install_location(_rclpy_action)
+clean_windows_flags(_rclpy_action)
 
 # Logging support provided by rcutils
 pybind11_add_module(_rclpy_logging SHARED

